clinica/iotools/bids_utils.py

```diff
@@ -401,6 +401,17 @@ def create_participants_df(
                     file_to_read = load_clinical_csv(
                         clinical_data_dir, location.split(".")[0]
                     )
+                    if location == "APOERES.csv":
```

Comment:
This is only for adni-to-bids right ? Maybe add something like:

```python
if study_name == StudyName.ADNI and location == "APOERES.csv":
    ...
```

Reply:
Yes this is for ADNI only ; the file is specific to ADNI too